benchmark-service/deployment.yaml

```diff
@@ -4,7 +4,12 @@ kind: Deployment
 metadata:
   name: benchmark
   labels:
+<<<<<<< HEAD
     app: web
+=======
+    app: benchmark
```

Comment:
Rebase needs to be fixed here.

Reply:
what do you mean?

Reply:
I meant the rebase insertions shouldn't be there. You want the contents below the `========` and remove the `<<<< HEAD` and `>>>>>> build.yaml changes`